src/Text/Pandoc/Readers/Org.hs

```diff
@@ -472,7 +483,7 @@ rawBlockContent (indent, blockType) = try $
   unlines . map commaEscaped <$> manyTill indentedLine blockEnder
  where
    indentedLine = try $ ("" <$ blankline) <|> (indentWith indent *> anyLine)
-   blockEnder = try $ indentWith indent *> stringAnyCase ("#+end_" <> blockType)
+   blockEnder = try $ skipSpaces *> stringAnyCase ("#+end_" <> blockType)
```

Comment:
Just for completeness:  There is little to no chance that indentation-based nesting gets added to Emacs org-mode. Quting from the [mailing list](https://lists.gnu.org/archive/html/emacs-orgmode/2016-05/msg00059.html):

>> Is there any chance that indentation-based nesting will become legit
>> org-mode syntax?
>
> I don't think so. It would introduce unnecessary complexity.

The '*'s in above example should be commas and those are already supported.  Not sure why I thought it should be an asterisk.